admin: SSL - copy cleanup changes from manager-5.0/4.3 (#3681) (#3682)
uyuni-project/uyuni-docs: modules/administration/pages/ssl-certs-imported.adoc

.Procedure: Import Certificates on a New {productname} Server

> . Deploy the {productname} Server according to the instructions in xref:installation-and-upgrade:install-server.adoc[].

Author: ```suggestion
Deploy the {productname} Server according to the instructions in xref:installation-and-upgrade:container-deployment/suma/server-deployment-suma.adoc[].
```

> Deploy the {productname} Server according to the instructions in xref:installation-and-upgrade:container-deployment/suma/server-deployment-suma.adoc[].

Author: ```suggestion
. Deploy the {productname} Server according to the instructions in xref:installation-and-upgrade:container-deployment/suma/server-deployment-suma.adoc[].
```

.Procedure: Import Certificates on a New {productname} Proxy

> . Install the {productname} Proxy according to the instructions in xref:installation-and-upgrade:install-proxy.adoc[].

Author: ```suggestion
. Install the {productname} Proxy according to the instructions in xref:installation-and-upgrade:container-deployment/suma/proxy-deployment-suma.adoc[].
```
